src/plugins/administrative_templates/ui/i18n/filteroptions_en.ts

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="en_US">
+<context>
+    <name>FilterOptionsDialog</name>
+    <message>
+        <location filename="../filteroptionsdialog.ui" line="14"/>
+        <source>Dialog</source>
+        <translation>Dialog</translation>
+    </message>
+</context>
+<context>
+    <name>FilterOptionsWidget</name>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="14"/>
+        <source>Form</source>
+        <translation>Form</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="50"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select options below to enable and change or disabletypes of global filters that will be applied to the Administrative Templates nodes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>Select options below to enable and change or disabletypes of global filters that will be applied to the Administrative Templates nodes.</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="91"/>
+        <source>Configured:</source>
+        <translation>Configured:</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="101"/>
+        <source>Managed:</source>
+        <translation>Managed:</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="127"/>
+        <location filename="../filteroptionswidget.ui" line="161"/>
+        <location filename="../filteroptionswidget.ui" line="202"/>
+        <location filename="../filteroptionswidget.ui" line="281"/>
+        <source>Any</source>
+        <translation>Any</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="132"/>
+        <location filename="../filteroptionswidget.ui" line="166"/>
+        <location filename="../filteroptionswidget.ui" line="207"/>
+        <location filename="../filteroptionswidget.ui" line="286"/>
+        <source>Yes</source>
+        <translation>Yes</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="137"/>
+        <location filename="../filteroptionswidget.ui" line="171"/>
+        <location filename="../filteroptionswidget.ui" line="212"/>
+        <location filename="../filteroptionswidget.ui" line="291"/>
+        <source>No</source>
+        <translation>No</translation>
+    </message>
+    <message>
+        <location filename="../filteroptionswidget.ui" line="179"/>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select the type of policy settings to display.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
```

Comment:
It will be cleaner if we disable rich text here.

Reply:
fixed